main.py

```diff
@@ -143,12 +160,12 @@ def _get_projects(client):
 
 
 def _get_resource_manager_client():
-    client = discovery.build("cloudresourcemanager", "v3")
+    client = discovery.build("cloudresourcemanager", "v3", cache_discovery=False)
```

Comment:
Can you please add a comment on why not caching discovery is needed now?

Reply:
This parameter was been removed.
This was added to suppress warnings of incompatibility between cache_discovery and oauth2 4.0.0, and in this project the oauth2's version used is 3.0.0
[a0575f9](https://github.com/kieras/zombie-projects-watcher/pull/63/commits/a0575f9e9f6fd42a9f4e79a69ac8b23040946d26)